Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp

```diff
@@ -232,9 +232,9 @@ void Wiimote::Reset()
   //   0x55 - 0xff: level 4
   m_status.battery = (u8)(m_battery_setting->GetValue() * 100);
 
-  memset(m_shake_step, 0, sizeof(m_shake_step));
-  memset(m_shakeSoft_step, 0, sizeof(m_shakeSoft_step));
-  memset(m_shakeHard_step, 0, sizeof(m_shakeHard_step));
+  memset(m_shake_step.data(), 0, sizeof(m_shake_step));
```

Comment:
You can just do `m_shake_step = {};` or `m_shake_step.fill(0);` instead of using memset now.